provider-anynines/internal/controller/serviceinstance/serviceinstance_test.go

```diff
@@ -997,7 +998,7 @@ func TestCreate(t *testing.T) {
 							OrganizationGUID:  "a1612e60-3042-4bf2-bd7c-fa600a4f66b9",
 							SpaceGUID:         "009dbe05-925d-4f2a-ac0d-8d44dd723a11",
 							Parameters: map[string]interface{}{
-								"max_connections": 100,
+								"max_connections": float64(100),
```

Comment:
Why use a `float64` type for `max_connections`? I would have expected an `int` type, i.e. `int32` since there are no "partial" connections.

Reply:
I feel like using an integer type for this value would be better but if the value is not processed by the controller and only forwarded to the broker, then it is OK to use `float64` for this value.